engine/src/main/java/io/zeebe/engine/processing/bpmn/BpmnStreamProcessor.java

```diff
@@ -147,6 +147,7 @@ private void processEvent(
       case ACTIVATE_ELEMENT:
         if (MigratedStreamProcessors.isMigrated(context.getBpmnElementType())) {
           final var activatingContext = stateTransitionBehavior.transitionToActivating(context);
+          stateTransitionBehavior.onElementActivating(element, activatingContext);
```

Comment:
:information_source:  We only need it for the migrated element processors. No need to change the logic for non-migrated processors.